Overview-of-the-Oppia-codebase.md

```diff
@@ -105,31 +103,19 @@ def handler(self, argument_1, argument_2):
 
 Controllers are stored in `core/controllers/`. `main.py` defines which controller should handle each request.
 
-### Backend service layer
-
-Services get called by controller functions or other services to perform various computations. They often need to access Oppia's stored data, which they do by calling functions in the domain and storage layers.  We'll look at those next.
-
-Services are, confusingly, defined by files in the `core/domain/` folder. Services end in `_services.py`.
-
-### Backend models
-
-Before we talk about the domain layer, we need to discuss models.
-
-A model is a data structure, often a class, that stores the information that defines a particular object in Oppia. For example, we have a model for each exploration. (Note that we use "model" to refer to both the class that defines the exploration model and each instance of that class. Sorry, we know it's confusing, but this language is all over the code base.)
-
-There are many different kinds of models in Oppia. We have frontend models, backend (domain) models, and storage models.
-
 ### Domain layer
```

Comment:
the backend architecture diagram shows separate domain and service layers. Should that be changed too?

Reply:
Thanks, good catch. They should still be separate, but on closer examination I think the original diagram was a bit wrong, so I simplified it.